linode/manager
# feat: [M3-6743] - Create Subnet Drawer

## Description 📝
- Add a create subnet drawer

## Preview 📷

https://github.com/linode/manager/assets/139280159/710eabd3-ab7b-43f5-a4b3-e60d36913538

## How to test 🧪
`yarn test CreateSubnetDrawer`
`yarn test SubnetNode`
`yarn cy:run -s "cypress/e2e/core/vpc/vpc-details-page.spec.ts"`
- On your dev account, navigate to a VPC's detail page and ensure that you can create a subnet / that appropriate errors show up
- make sure there are no visual changes to VPCCreate (which also uses SubnetNode)
